sites/game-client/map-renderer.mjs

```diff
@@ -317,6 +317,10 @@ class MapRenderer {
    * @param {any} renderArgs - Optional. The optional argument if renderFunction requires one.
    */
   registerCustomizedLayerToDraw(zIndex, layerName, renderFunction, renderArgs) {
+    console.assert(renderFunction !== '_drawOneCharacterImage' ||
+                     renderArgs !== null,
+                   'No render args for _drawOneCharacterImage',
+                   [zIndex, layerName, renderFunction, renderArgs]);
```

Comment:
`renderFunction` is not necessarily `_drawOneCharacterImage`, this assert does not make sense to me.